Contact.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+class Contact
+{
+
+	// variables
+	public $ID=null;
+	public $FirstName = '';
+	public $LastName = '';
+	public $Email = '';
+
+
+	// setter
+	public function setValue($field,$newValue)
+	{
+		$this->$field=$newValue;
+		
+	} 
+
+	// getter
+	public function getValue($field)
+	{
+		return $this->$field;
+	}
+
+	// push Contact info into the db.
+	public function save()
+	{
+		$config = parse_ini_file('user.credentials.ini');
```

Comment:
If you save multiple times within a lifetime of a script you'll end up parsing this file over and over.. probably want to store the connection as a static global for the class. ANd then when it hits save, it see's if it exists, if not then connect.